### DISCUSSION TITLE
Downgraded Galaxy Buds 2 Pro

### DISCUSSION THREAD

**Discussion by danny9090** [timschneeb/GalaxyBudsClient#400] [Q&A]
Hello today i connected my Galaxy Buds 2 Pro with my PC and Downgraded the Firmware to the first on the List after that i cant connect my Galaxy Buds 2 Pro anymore can anyone please Help me :(

**Comment by Appfanatic456** [comment]
Try upgrading the firmware - remember that the Galaxy Buds 2 Pro are new technology. So they can easier connect with the latest systems.
